### PR TITLE
chore: add page for tracking Borceux progress

### PR DESCRIPTION
**Pull request by TOTBWF**
# Description

Adds a page for tracking progress on Borceux. Somewhat surprisingly, we are already 34% of the way done!

## Checklist

Before submitting a merge request, please check the items below:

- [x] I've read [the contributing guidelines](https://github.com/plt-amy/1lab/blob/main/CONTRIBUTING.md).
- [x] The imports of new modules have been sorted with `support/sort-imports.hs` (or `nix run --experimental-features nix-command -f . sort-imports`).
- [x] All new code blocks have "agda" as their language.

If your change affects many files without adding substantial content, and
you don't want your name to appear on those pages (for example, treewide
refactorings or reformattings), start the commit message and PR title with `chore:`.

### CODE REVIEW

**Comment by Lavenza** [comment]
[Pull request preview](https://preview.1lab.dev/402)

File: src/Borceux.lagda.md [hunk 1]
```diff
@@ -0,0 +1,1443 @@
...
+* Proposition 1.2.4: `Strict-cats`{.Agda}
+* Examples 1.2.5:
+  * a. `Sets`{.Agda}
+  * b. ❌
+  * c. `Groups`{.Agda}
+  * d. ❌
+  * e. ❌
+  * f. ❌
+  * g. ❌
+  * h. ❌
```
Comment (plt-amy) [inline src/Borceux.lagda.md:157]: Do we need these trailing :x:?

Comment (TOTBWF) [reply src/Borceux.lagda.md:157]: I find it useful to know what hasn't been done so that I know what to work on; I'm open to other emojis though!

Comment (plt-amy) [reply src/Borceux.lagda.md:157]: Wouldn't it be better to have these in a todo list in an issue or something?

Comment (plt-amy) [reply src/Borceux.lagda.md:157]: Or a [project](https://github.com/plt-amy/1lab/projects?query=is%3Aopen)

Comment (TOTBWF) [reply src/Borceux.lagda.md:157]: Fair enough; I'll open a project and take some times to categorize things a bit more

File: src/Borceux.lagda.md [hunk 2]
```diff
@@ -0,0 +1,939 @@
...
+* Proposition 2.9.5: `representable-reverses-colimits`{.Agda}
+* Definition 2.9.6: `reflects-limit`{.Agda}
+* Proposition 2.9.7: `conservative-reflects-limits`{.Agda}
+
+### 2.10 Absolute colimits
+
+### 2.11 Final functors
+
+::: warning
+Borceux uses some outdated terminology here , and also uses a condition
```
Comment (plt-amy) [inline src/Borceux.lagda.md:532]:
```suggestion
Borceux uses some outdated terminology here, and also uses a condition
```

File: src/Borceux.lagda.md [hunk 3]
```diff
@@ -0,0 +1,939 @@
...
+-->
+
+* Definition 2.1.1: `is-product`{.Agda}
+* Proposition 2.1.2: `×-Unique`{.Agda}
+* Proposition 2.1.3:
+  * 1. `Binary-products.swap-is-iso`{.Agda}
+  * 2. `Cartesian-monoidal`{.Agda}
+* Definition 2.1.4: `is-indexed-product`{.Agda}
+* Proposition 2.1.5: `Indexed-product-unique`{.Agda}
+* Examples 2.1.7:
```
Comment (plt-amy) [inline src/Borceux.lagda.md:391]: Should this still be here?

File: src/Borceux.lagda.md [hunk 4]
```diff
@@ -0,0 +1,939 @@
...
+
+<!--
+```agda
+_ = Hom[-,-]
+```
+-->
+
+* Definition 1.10.1: `_^op`{.Agda}
+* Examples 1.10.3:
+  * a. `Hom[-,-]`
```
Comment (plt-amy) [inline src/Borceux.lagda.md:351]:
```suggestion
  * a. `Hom[-,-]`{.Agda}
```

File: src/Borceux.lagda.md [hunk 5]
```diff
@@ -0,0 +1,939 @@
...
+
+<!--
+```agda
+_ = よ-preserves-mono
+_ = よcov-reverses-epi
+_ = has-section+monic→invertible
+```
+-->
+
+* Exercise 1.11.1: 🚧 `thin-functor`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:363]: broken link

File: src/Borceux.lagda.md [hunk 6]
```diff
@@ -0,0 +1,939 @@
...
+* Exercise 1.11.5: `よ-preserves-mono`{.Agda}
+* Exercise 1.11.6: `よcov-reverses-epi`{.Agda}
+* Exercise 1.11.8: 🚧 `Curry`{.Agda}, `Uncurry`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:366]: broken link

File: src/Borceux.lagda.md [hunk 7]
```diff
@@ -0,0 +1,939 @@
...
+-->
+
+* Definition 2.5.1: `is-pullback`{.Agda}
+* Proposition 2.5.2: `Pullback-unique`{.Agda}
+* Proposition 2.5.3:
+  * 1. `is-monic→pullback-is-monic`{.Agda}
+* Proposition 2.5.7: `is-effective-epi.is-effective-epi→is-regular-epi`{.Agda}
+* Proposition 2.5.8: `is-regular-epi→is-effective-epi`{.Agda}
+* Proposition 2.5.9:
+  * 1. `pasting-left→outer-is-pullback`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:453]: broken link

File: src/Borceux.lagda.md [hunk 8]
```diff
@@ -0,0 +1,939 @@
...
+* Examples 2.5.10
+  * a. `Sets-pullbacks`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:455]: broken link

File: src/Borceux.lagda.md [hunk 9]
```diff
@@ -0,0 +1,939 @@
...
+### 2.17 Exercises
+
+<!--
+```agda
+_ = Cone→cone
+```
+-->
+
+* Exercise 2.17.3: 🚧 `Cone→cone`{.Agda}
+* Exercises 2.17.8: `extend-is-colimit`{.Adga}, `is-colimit-restrict`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:588]:
```suggestion
* Exercises 2.17.8: `extend-is-colimit`{.Agda}, `is-colimit-restrict`{.Agda}
```

File: src/Borceux.lagda.md [hunk 10]
```diff
@@ -0,0 +1,939 @@
...
+_ = is-counit-iso→is-reflective
+_ = is-equivalence
+```
+-->
+
+* Proposition 3.4.1:
+  (⇒). `is-reflective→counit-is-iso`{.Agda}
+  (⇐). `is-counit-iso→is-reflective`{.Agda}
+
+* Definition 3.4.4: `is-equivalence`{.Adga}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:647]:
```suggestion
* Definition 3.4.4: `is-equivalence`{.Agda}
```

File: src/Borceux.lagda.md [hunk 11]
```diff
@@ -0,0 +1,939 @@
...
+
+* Definition 3.7.1: `is-lan`{.Agda}
+* Theorem 3.7.2: `cocomplete→lan`{.Agda}
+* Proposition 3.7.3: `ff→pointwise-lan-ext`{.Agda}
+* Proposition 3.7.4: `left-adjoint→left-extension`{.Agda}
+* Proposition 3.7.5:
+  * (⇒) `is-initial-cocone→is-colimit`{.Agda}
+  * (⇐) `is-colimit→is-initial-cocone`{.Agda}
+* Proposition 3.7.6:
+  * (1 ⇒ 2) `adjoint→is-lan-id`{.Agda}, `adjoint→is-absolute-lan`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:685]: first link here is broken

File: src/Borceux.lagda.md [hunk 12]
```diff
@@ -0,0 +1,939 @@
...
+```agda
+_ = right-adjoint→objectwise-rep
+_ = corepresentable→left-adjoint
+_ = Karoubi-is-completion
+```
+-->
+
+* Exercise 3.9.2:
+  * (⇒) `corepresentable→left-adjoint`{.Agda}
+  * (⇐) `right-adjoint→objectwise-rep`
```
Comment (plt-amy) [inline src/Borceux.lagda.md:701]:
```suggestion
  * (⇐) `right-adjoint→objectwise-rep`{.Agda}
```

File: src/Borceux.lagda.md [hunk 13]
```diff
@@ -0,0 +1,939 @@
...
+_ = in-subcategory→orthogonal-to-inverted
+_ = orthogonal-to-ηs→in-subcategory
+_ = in-subcategory→orthogonal-to-ηs
+```
+-->
+
+* Definition 5.4.1: `m⊥m`{.Agda}
+* Definition 5.4.2:
+  1. `m⊥o`
+  2. `o⊥m`
```
Comment (plt-amy) [inline src/Borceux.lagda.md:793]:
```suggestion
  1. `m⊥o`{.Agda}
  2. `o⊥m`{.Agda}
```

File: src/Borceux.lagda.md [hunk 14]
```diff
@@ -0,0 +1,939 @@
...
+_ = factorisation-essentially-unique
+_ = E-is-⊥M
+```
+-->
+
+* Definition 5.5.1: `is-factorisation-system`{.Agda}
+* Proposition 5.5.2: `factorisation-essentially-unique`{.Agda}
+* Proposition 5.5.3: 🚧 `E-is-⊥M`{.Agda}
+* Proposition 5.5.4:
+  * 3. `in-intersection≃is-iso`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:814]: broken link

File: src/Borceux.lagda.md [hunk 15]
```diff
@@ -0,0 +1,939 @@
...
+_ = _=>o_
+_ = Outer-functors
+_ = ConstO
+```
+-->
+
+* Definition 8.2.1: `Outer-functor`{.Agda}
+* Definition 8.2.2: `_=>o_`{.Agda}
+* Proposition 8.2.3: `Outer-functors`{.Agda}
+* Example 8.2.4: `ConstO`{.Agda}, `const-nato`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:932]: `const-nato` is missing

**Review by plt-amy — CHANGES_REQUESTED** [review]

File: src/Borceux.lagda.md [hunk 16]
```diff
@@ -0,0 +1,939 @@
...
+* Definition 1.3.1: `_=>_`{.Agda}
+* Proposition 1.3.2: `Cat[_,_]`{.Agda}
+* Theorem 1.3.3:
+  * 1. `yo-is-equiv`{.Agda}
+  * 2. `yo-naturalr`{.Agda}
+  * 3. `yo-naturall`{.Agda}
+* Proposition 1.3.4: `_◆_`{.Agda}
+* Examples 1.3.6:
+  * c. `よcov₁`{.Agda}
+  * d. `const-nt`{.Agda}
```
Comment (plt-amy) [inline src/Borceux.lagda.md:184]: this one's missing too

**Review by plt-amy — APPROVED** [review]

**Pull request merged**